core/src/main/java/it/pagopa/selfcare/mscore/core/EmailService.java

```diff
@@ -47,27 +46,28 @@ public void sendAutocompleteMail(List<String> destinationMail, Map<String, Strin
         emailConnector.sendMail(mailTemplateConfig.getAutocompletePath(), destinationMail, file, productName, templateParameters, fileName);
     }
 
-    public void sendMail(File pdf, Institution institution, User user, OnboardingRequest request, String token, boolean isApproved, InstitutionType institutionType) {
+    public void sendMailWithContract(File pdf, Institution institution, User user, OnboardingRequest request, String token) {
```

Comment:
you could avoid to pass Institution object if you use only institution.digitalAdress....you could replace institution attribute with String digitalAddress in method signature